contrib/cf-remote/cf_remote/commands.py

```diff
@@ -1,13 +1,22 @@
 import os
 import sys
+import time
 
 from cf_remote.remote import get_info, print_info, install_host, run_command, transfer_file
 from cf_remote.packages import Releases
 from cf_remote.web import download_package
 from cf_remote.paths import cf_remote_dir
-from cf_remote.utils import save_file, strip_user
+from cf_remote.utils import save_file, strip_user, read_json, write_json
+from cf_remote.spawn import VM, VMRequest, Providers, AWSCredentials
+from cf_remote.spawn import spawn_vms, destroy_vms, dump_vms_info, get_cloud_driver
 from cf_remote import log
+from cf_remote import cloud_data
 
+CF_REMOTE_DATA_DIR = os.path.expanduser("~/.cfengine/cf-remote")
+CLOUD_CONFIG_FNAME = "cloud_config.json"
+CLOUD_CONFIG_FPATH = os.path.join(CF_REMOTE_DATA_DIR, CLOUD_CONFIG_FNAME)
+CLOUD_STATE_FNAME = "cloud_state.json"
+CLOUD_STATE_FPATH = os.path.join(CF_REMOTE_DATA_DIR, CLOUD_STATE_FNAME)
```

Comment:
I don't like them being functions. Why not just constants? (in the *paths* module)

Reply:
That's fine as well. For directories I prefer:

```
cf_remote_dir("packages.json")
```

instead of:

```
os.path.join(CF_REMOTE_DIR, "packages.json")
```